app/views/controllers/inat_imports/show.html.erb

```diff
@@ -31,3 +31,10 @@
     { class: "mt-3 btn btn-default" }
   )
 %>
+<%=
+  link_to(
+    :inat_import_tracker_cancel.l,
+    inat_import_cancel_path(id: @tracker.inat_import),
+    { class: "mt-3 btn btn-default" }
+  )
```

Comment:
This will need `data: { turbo_stream: true }`. `class` and `data` do not need to be inside a hash.

Reply:
Also I'd strongly vote for using the translation string `:CANCEL.l` instead of MO's unfortunate practice of proliferating duplicate translation strings. We are eventually going to have to weed out all the duplicates if we are ever going to use JS front-end translation, because the data size for all these keys and values is unmanageable.